packages/kit/test/apps/basics/src/routes/__layout.svelte

```diff
@@ -9,6 +11,13 @@
 			}
 		};
 	}
+
+	if (browser) {
+		let h3 = document.createElement('h3');
+		let text = document.createTextNode('You cannot reset me');
+		h3.appendChild(text);
+		document.getElementById('nested-layout-reset-test')?.appendChild(h3);
```

Comment:
I would expect that `?.` fails on Node 12 on the CI. For some very strange reason, the CI did not appear to run on this PR. Github was having issues earlier today. Perhaps if you push any change (such as changing to `const` as suggested above) it will run the CI